cmd/kpod/pull.go

```diff
@@ -0,0 +1,146 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/Sirupsen/logrus"
+	cp "github.com/containers/image/copy"
+	"github.com/containers/image/docker/reference"
+	"github.com/containers/image/signature"
+	is "github.com/containers/image/storage"
+	"github.com/containers/image/transports/alltransports"
+	"github.com/containers/image/types"
+	"github.com/containers/storage"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+const (
+	// DefaultRegistry is a prefix that we apply to an image name
+	// to check docker hub first for the image
+	DefaultRegistry = "docker://"
+)
+
+// PullOptions is used to initialize and store the information received
+// from the command line
+type PullOptions struct {
+	FromImage           string
+	AllTags             bool
+	DisableContentTrust bool
+}
+
+var (
+	pullFlags = []cli.Flag{
+	// cli.BoolFlag{
+	// 	Name:  "all-tags, a",
+	// 	Usage: "Download all tagged images in the repository",
+	// },
+	// cli.BoolFlag{
+	// 	Name:  "disable-content-trust",
+	// 	Usage: "Skip image verification",
+	// },
+	}
+
+	pullDescription = "Pull an image from a registry"
+	pullCommand     = cli.Command{
+		Name:        "pull",
+		Usage:       "pull an image from a registry",
+		Description: pullDescription,
```

Comment:
Add some other data besides pullDescription.  This ends up with the --help repeating itself.
Grab some description from the man page.

`Copies an image from a registry onto the local machine. **kpod pull** pulls an
image from Docker Hub if a registry is not specified in the command line argument.
If an image tag is not specified, **kpod pull** defaults to the image with the
**latest** tag (if it exists) and pulls it. **kpod pull** can also pull an image
using its digest **kpod pull [image]@[digest]**.